_posts/platform/app/2000-01-01-run-scheduled-tasks.md

```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
 | */10 * * * * echo biniou |      |
 +--------------------------+------+
 ```
+
+## Delete Configured Cron Tasks
+
+In order to delete a cron tasks, you must delete the associated entry in the `cron.json`.
+If there is no more tasks, or if you want to remove all tasks, you can also remove the file.
+
+The cron tasks configuration will be updated when redeploying the application.
+
+## FAQ
+
+### How To Enable Cron Tasks?
+
+The cron feature is enabled by default.
+
+### How To Disable Cron Tasks?
+
+You may want to disable automatic cron tasks execution, for example for [review apps]({% post_url
+platform/app/2000-01-01-review-apps %}) or staging environments.
+
+TBD
+
+### How Precise Is The Cron Feature?
+
+Since the cron feature needs to start a new container to execute the task, the precision will depend on the
+deployment time.
+
+### How Will I Be Billed?
+
+The cron feature in itself is free. The one-off containers launched to run the commands defined in your cron tasks will
+be like other [one-off containers]({% post_url platform/app/2000-01-01-tasks %}) ie you will be billed for the type of
+container you defined in you cron task (M is the default container size) for the time it was executed. For example of
```

Comment:
This sentence is too long and should be split in shorter sentences.

"ie" is spelled "i.e."

We shouldn't use past tense in documentation